devenv/setenv.sh

```diff
@@ -0,0 +1,19 @@
+export B2ACCESS_CONSUMER_KEY="<B2ACCESS_USERNAME_FOR_OAUTH2_CLIENT>"
+export B2ACCESS_SECRET_KEY="<B2ACCESS_PASSWORD_FOR_OAUTH2_CLIENT>"
+export B2SHARE_ACCOUNTS_SESSION_REDIS_URL="redis://localhost:6379/0"
+export B2SHARE_BROKER_URL="redis://localhost:6379/0"
+export B2SHARE_CELERY_RESULT_BACKEND="redis://localhost:6379/1"
```

Comment:
B2SHARE_ACCOUNTS_SESSION_REDIS_URL, B2SHARE_BROKER_URL and B2SHARE_CELERY_RESULT_BACKEND are not necessary. I think these are already the default values.